aio/aio-proxy/aio_proxy/response/build_response.py

```diff
@@ -5,31 +5,33 @@
 from aio_proxy.decorators.http_exception import http_exception_handler
 from aio_proxy.response.format_response import format_response
 from aio_proxy.response.format_search_results import format_search_results
-from aio_proxy.search.index import ElasticsearchSireneIndex
+from aio_proxy.search.es_search_runner import ElasticSearchRunner
 from aiohttp import web
 from sentry_sdk import capture_exception, push_scope
 
 
 @http_exception_handler
 def api_response(
-    request, extract_function: Callable, search_function: Callable
+    request,
+    extract_function: Callable,
```

Comment:
I still think that extract function should not be passed as an arguments but determined internally as can be determined from `search_type`

Reply:
I had planned to do so in the following PR but I ended up adding it here as well.